### DISCUSSION TITLE
Liaison Utilisateur/Adresse mail

### DISCUSSION THREAD

**Discussion by KenneyClement** [witsa/synapps#802] [Ideas]
lorsqu'on fait une liaison Utilisateur/Adresse mail sur un acteur on a pas la possibilité d'écrire 
![image](https://github.com/user-attachments/assets/de502487-2766-470b-a98f-650d4264613b)

**Comment by Mathis554** [comment]
Bonjour,
Il faut que tu coche "Ecriture ?"

**Reply by WITNicolas** [reply]
Bonjour,

Complément de réponse

Attention au niveau des utilisateurs, seul le niveau Administrateur est autorisé à écrire dans ces champs.
Les niveaux inférieurs n'accédent pas au menu CONFIGURATION du REDY.

**Reply by Mathis554** [reply]
Oh effectivement

**Reply by Mathis554** [reply]
[Utilisateur.zip](https://github.com/user-attachments/files/18349703/Utilisateur.zip)
Voici une prog d'exemple
